### PR TITLE
misc: Add a few more functions to auto args

### PR DESCRIPTION
**Pull request by honggyukim**
This PR adds a few more functions to auto args list.

### CODE REVIEW

**Comment by namhyung** [comment]
Hmm.. I guess these `_IO_` variants might be used for other functions too?  Also I think it'd be better updating necessary functions at once after you collect them for a while.

**Comment by honggyukim** [comment]
Okay. Then I will not add `_IO_` variants at this time but I need others when analyzing Qt program at the moment. If you're okay with it l will update it again.

File: misc/prototypes.h [hunk 1]
```diff
@@ -218,6 +218,8 @@ enum uft_dlopen_flag {
 	RTLD_NODELETE = 0x1000,
 };
 void *dlopen(const char *filename, enum uft_dlopen_flag flags);
+void *dlsym(void *handle, const char *symbol);
+void *dlvsym(void *handle, char *symbol, char *version);
```
Comment (namhyung) [inline misc/prototypes.h]: Wouldn't it be better adding `dlclose()` and `dlerror()` as well?

File: misc/prototypes.h [hunk 2]
```diff
@@ -44,6 +44,26 @@ void *mmap64(void *addr, size_t length, enum uft_mmap_prot prot, enum uft_mmap_f
...
+    MADV_MERGEABLE   = 12,
+    MADV_UNMERGEABLE = 13,
+    MADV_HUGEPAGE    = 14,
+    MADV_NOHUGEPAGE  = 15,
+    MADV_DONTDUMP    = 16,
+    MADV_DODUMP      = 17,
+    MADV_HWPOISON    = 100,
+};
+int madvise(void *addr, size_t length, enum uft_madvise_advice advice);
+
```
Comment (namhyung) [inline misc/prototypes.h]: There are `posix_madvise` and `posix_fadvise` too.

**Comment by honggyukim** [comment]
Hi @namhyung, I've updated as you mentioned. Please take a look at it!

File: misc/prototypes.h [hunk 3]
```diff
@@ -44,6 +44,45 @@ void *mmap64(void *addr, size_t length, enum uft_mmap_prot prot, enum uft_mmap_f
...
+enum uft_posix_madvise_advice {
```
Comment (namhyung) [inline misc/prototypes.h]: Sorry for bothering you, but the name seems too long.  What about `uft_posix_madvise` or `uft_posix_madvise_adv`?

Comment (honggyukim) [reply misc/prototypes.h]: Ok. I will make it as `uft_posix_madvise_adv` then.

Comment (honggyukim) [reply misc/prototypes.h]: Just updated it. Thanks!

File: misc/prototypes.h [hunk 4]
```diff
@@ -44,6 +44,45 @@ void *mmap64(void *addr, size_t length, enum uft_mmap_prot prot, enum uft_mmap_f
 int munmap(void *addr, size_t length);
 int mprotect(void *addr, size_t len, enum uft_mmap_prot prot);
 
+enum uft_madvise_advice {
```
Comment (namhyung) [inline misc/prototypes.h]: This also can be just `uft_madvise`.

Comment (honggyukim) [reply misc/prototypes.h]: Sure. Also updated it!

**Pull request merged**